utils/payments.ts

```diff
@@ -1,4 +1,7 @@
 // Copyright 2023 the Deno authors. All rights reserved. MIT license.
+
+// Default types for Stripe don't yet work: https://github.com/stripe-samples/stripe-node-deno-samples/issues/2
+// @deno-types="npm:stripe"
```

Comment:
Can you move this to a `stripe.ts` file at the root of this codebase and point the `stripe` import in the `import_map.json` to that file? That way, all files that import `stripe` also get types.

Also, can you please add a version that ensures the types and runtime versions of `stripe` match?